src/configure-nilrt-snac

```diff
@@ -188,6 +188,30 @@ EOF
 	rmmod cfg80211 mac80211 >/dev/null 2>&1
 }
 
+configure_tmux() {
+	log INFO Install tmux
+	opkg update >/dev/null && opkg install tmux
+
+	log INFO Create timeout configuration for tmux and source it
+	mkdir -p /usr/share/nilrt-snac
+	cat <<EOF >/usr/share/nilrt-snac/tmux-snac.conf
+# NILRT SNAC configuration tmux-snac.conf. Do not hand-edit.
+
+set -g lock-after-time 900
+EOF
```

Comment:
FYI: You can use `<<-EOF` (with a minus-sign) to direct bash to ignore leading whitespace in your heredoc. Then you can indent the heredoc without the indents carrying through.